source/core/ut_suite_builder.pkb

```diff
@@ -583,6 +625,30 @@ create or replace package body ut_suite_builder is
     a_suite.path := lower(coalesce(a_suite.path, a_suite.object_name));
   end;
 
+  procedure add_tags_to_suite(
```

Comment:
Seems like duplicate with `add_tags_to_test`
Any way we can make that function more generic?
Maybe we should have `add_tags_to_suite_item`?

Reply:
Yes no that we moved tags into suite_items it makes sense